src/Numerical/Spectral/Linearize.hpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function Linearize.
+
+#pragma once
+
+#include <cstddef>
+
+class DataVector;
+template <size_t>
+class Index;
+template <size_t Dim>
+using Mesh = Index<Dim>;
+
+/*! \ingroup Functors
+ *
+ * Truncate u to a linear function in each dimension.
+ * Ex in 2D: u^Lin = U_0 + U_x x + U_y y + U_{xy} xy
```

Comment:
I think this needs to be marked as math somehow, but I don't remember how offhand.